exonum/src/blockchain/schema.rs

```diff
@@ -120,8 +124,29 @@ impl<T: Access> Schema<T> {
     ///
     /// This method can be used to retrieve a proof that a certain transaction
     /// result is present in the blockchain.
-    pub fn transaction_results(&self) -> ProofMapIndex<T::Base, Hash, ExecutionStatus> {
-        self.access.clone().get_proof_map(TRANSACTION_RESULTS)
+    pub fn call_errors(
+        &self,
+        block_height: Height,
+    ) -> ProofMapIndex<T::Base, CallLocation, ExecutionError> {
```

Comment:
Well, the error descriptions are stored for now and are sort of useful for debug purposes at least. Since error descriptions do not influence `error_hash`, it could be possible to remove them on some schedule if the storage usage is a concern.

Reply:
Makes sense.